### ISSUE TITLE
I2I: Add `tabindex` attribute to `amp-iframe`

### ISSUE THREAD

**Issue by sebil** [ampproject/amphtml#23478]
## Summary
In order to have full control over accessing an iframe via keyboard use, it should be possible to set the `tabindex` attribute in `amp-iframe`.

## Motivation
When I tried to make the **amp.dev home page** more accessible by keyboard, I found out that the user gets stuck inside the **code-preview** of the "AMPHTML Playground" section, which is implemented as an iframe.

Giving the `amp-iframe` element a `tabindex="-1"` to **disable focusing** didn't work, because the attribute won't be applied to the actual `iframe` element inside `amp-iframe`.

The biggest problem is that the user can't get out of the code-preview by using tabs, because the tab-key is mapped for indention in the code-editor. The only exception is with `option/alt + tab`.

## Additional context
For me, it's important to implement this feature, because otherwise the developer is limited in using basic html functionality.

/cc @ampproject/wg-approvers
